spec/features/hq/orphan_spec.rb

```diff
@@ -0,0 +1,163 @@
+    fill_in 'orphan[family_name]', with: 'Asdf'
+    check 'orphan[father_deceased]'
+    fill_in 'orphan[mother_name]', with: 'Aisha'
+    check 'orphan[mother_alive]'
+    check 'orphan[father_is_martyr]'
+    fill_in 'orphan[father_occupation]', with: 'Merchant'
+    fill_in 'orphan[father_place_of_death]', with: 'Gaza'
+    fill_in 'orphan[father_cause_of_death]', with: 'natural'
+    fill_in 'orphan[father_date_of_death]', with: '2015-06-01'
+    fill_in 'orphan[guardian_name]', with: 'Goku'
+    fill_in 'orphan[guardian_relationship]', with: 'sister'
+    fill_in 'orphan[guardian_id_num]', with: '123'
+    fill_in 'orphan[contact_number]', with: '321123'
+    fill_in 'orphan[alt_contact_number]', with: '000999'
+    fill_in 'orphan[original_address_attributes][city]', with: 'Rome'
+    select 'Hama', :from => 'orphan[original_address_attributes][province_id]'
+    fill_in 'orphan[original_address_attributes][neighborhood]', with: 'neighborhood'
+    fill_in 'orphan[original_address_attributes][street]', with: 'infinite loop 1'
+    fill_in 'orphan[original_address_attributes][details]', with: 'Lorem Ipsum'
+    fill_in 'orphan[current_address_attributes][city]', with: 'New Town'
+    select 'Homs', :from => 'orphan[current_address_attributes][province_id]'
+    fill_in 'orphan[current_address_attributes][neighborhood]', with: 'NewTownHood'
+    fill_in 'orphan[current_address_attributes][street]', with: 'NewTownStreet'
+    fill_in 'orphan[current_address_attributes][details]', with: 'NewTown details'
+    check 'orphan[sponsored_by_another_org]'
+    fill_in 'orphan[minor_siblings_count]', with: '10'
+    fill_in 'orphan[sponsored_minor_siblings_count]', with: '5'
+    fill_in 'orphan[comments]', with: 'This is a comment'
+    and_i_click_button "Update Orphan"
+    and_i_should_be_on "hq_orphan_path", { orphan_id: 1 }
+    and_i_should_see "Orphan successfuly saved"
+    and_i_should_see "Orphan N"
+    and_i_should_see "01 January 2010"
+    and_i_should_see 'Good'
+    and_i_should_see 'goes to school'
+    and_i_should_see 'Active'
+    and_i_should_see 'High'
+    and_i_should_see 'Orphan Nxy'
+    and_i_should_see 'Asdf'
+    and_i_should_see 'Aisha'
+    and_i_should_see 'Merchant'
+    and_i_should_see 'Gaza'
+    and_i_should_see 'natural'
+    and_i_should_see '01 June 2015'
+    and_i_should_see 'Goku'
+    and_i_should_see 'sister'
+    and_i_should_see 'Rome'
+    and_i_should_see '123'
+    and_i_should_see '321123'
+    and_i_should_see '000999'
+    and_i_should_see 'Hama'
+    and_i_should_see 'neighborhood'
+    and_i_should_see 'infinite loop 1'
+    and_i_should_see 'Lorem Ipsum'
+    and_i_should_see 'New Town'
+    and_i_should_see 'Homs'
+    and_i_should_see 'NewTownHood'
+    and_i_should_see 'NewTownStreet'
+    and_i_should_see 'NewTown details'
+    and_i_should_see 'Sponsored by Another Org. Yes'
+    and_i_should_see '10'
+    and_i_should_see '5'
+    and_i_should_see 'This is a comment'
+  end
```

Comment:
Uncle Bob would not like a method like this :) It is too long and it is doing too many things.
Can we extract some parts in new methods?

Like:
```
visit_orphan( "Orphan 1" )
and_i_click_link "Edit Orphan"
and_i_should_be_on "edit_hq_orphan_path", { orphan_id: 1 }

expect_filled_formfields_for "Orphan 1"  #checks that all formfields have the correct values

update_formfields_for "Orphan 1"  #fills in formfields with new data
and_i_click_button "Update Orphan"

and_i_should_be_on "hq_orphan_path", { orphan_id: 1 }
and_i_should_see "Orphan successfuly saved"
expect_updated_data_for "Orphan 1"  #checks that the new updated data is present
```